docker/GPU/Dockerfile

```diff
@@ -12,6 +12,17 @@ ENV FORCE_CUDA="1"
 
 ENV DEBIAN_FRONTEND=noninteractive
 
+### change the system source for install libs
+ARG USE_SRC_INSIDE
+RUN if [ -z ${USE_SRC_INSIDE}] ; \
+    then \
+        echo "Use official source for install libs" ; \
```

Comment:
How about `Keep the download source unchanged`

Reply:
it seems more reasonable